pom.xml

```diff
@@ -212,6 +127,13 @@
     <maven.build.timestamp.format>yyyyMMdd-HHmm</maven.build.timestamp.format>
     <jacoco-maven-plugin.version>0.7.6.201602180812</jacoco-maven-plugin.version>
     <mockito.version>1.10.19</mockito.version>
+    
+    <final.war.name>cbioportal-${project.version}</final.war.name>
+
+	<db.test.driver>com.mysql.jdbc.Driver</db.test.driver>
+	<db.test.url>jdbc:mysql://127.0.0.1:3306/cgds_test</db.test.url>
+    <db.test.username>cbioportal</db.test.username>
+    <db.test.password>cbioportal</db.test.password>
```

Comment:
Using the (1) pom.xml is the source for testing and for DB initialization, (2) and (3) are only for runtime. And they're bad ways to do that anyway. (1) is needed for the build process. I'm planning to get to a point where we don't need to use portal.properties for the build *at all*. This is part of that.